src/includes/proc-service.h

```diff
@@ -177,6 +177,9 @@ class base_process_service : public service_record
 
     log_type_id log_type = log_type_id::NONE;
     string logfile;           // log file name, empty string specifies /dev/null
+    int logfile_perms = 0;         // logfile permissions("mode")
+    uid_t logfile_uid = getuid();  // logfile user id or dinit daemon uid
```

Comment:
Don't initialise these here via getuid() etc, that's wasteful, they will be set via `set_logfile_details` when it is called. You can leave them as -1 here.

The comment should describe the purpose of the variable. It currently says "logfile user id _or_ dinit daemon uid" but the variable is _always_ for the logfile uid, the daemon uid is just the default value.